community/kernel/src/test/java/org/neo4j/kernel/impl/newapi/MockStore.java

```diff
@@ -579,12 +579,24 @@ void node( NodeRecord record, long reference, PageCursor pageCursor )
         initialize( record, reference, nodes );
     }
 
+    @Override
+    void nodeAdvance( NodeRecord record, PageCursor pageCursor )
+    {
+        initialize( record, record.getId() + 1, nodes );
```

Comment:
will this work for cases when we do not allocate ids one after another?

Reply:
well if this _MockStore_ suddenly breaks I guess you get to blame me :)